doc/en/logging.rst

```diff
@@ -198,6 +198,16 @@ option names are:
 * ``log_file_format``
 * ``log_file_date_format``
 
+You can call ``set_log_filename()`` to customize log_file path dinamically::
```

Comment:
```suggestion
You can call ``set_log_filename()`` to customize the log_file path dynamically::
```

Reply:
Thank you for the review. The typo is fixed, and also commits are squashed and rebased.